fulp_modules/antag_tips/code/antag_tip_integration.dm

```diff
@@ -0,0 +1,29 @@
+/datum/antagonist/traitor/finalize_traitor()
```

Comment:
You probably want to put just "/datum/antagonist/proc_name" there, otherwise it will only show up for traitors and subtypes.